### DISCUSSION TITLE
OsmAnd Pro subscription via F-Droid version without Google

### DISCUSSION THREAD

**Discussion by frederikb96** [osmandapp/OsmAnd#16313] [Q&A] [answered]
Hi,
I would like to subscribe to osmand Pro but I do not use Google and thus I can not subscribe via Google Play (or Amazon or Huawei). Is there another way to buy the subscription?

If there is none, would it be possible to buy it on another dummy phone with Google and restore it via the email address on the real phone?

Thanks!

**Comment by vshcherb** [comment, marked as answer]
PRO subscription is across all platforms, so once purchased on iPhone, Amazon or Huawei you can enable it on F-Droid etc... You just need to enter same account on OsmAnd Cloud and purchase will be restored

**Reply by CHoeb** [reply]
I use osmand~ from f-droid in /e/os (degoogled android), how can I use the cloud sync ?

**Reply by kemiaitesok** [reply]
Hi, I don't understand something.
I do not have a google account, my phone is running e/OS/. I install the OsmAND app from F-droid.
How do I buy from google play store if I don't have my own gmail address? I can't shop in google play with someone else's profile, email address. 
I deleted my old gmail address about 3 years ago, now I have a protonmail email address.
Is there no way to buy unlimited access on one site? I would have to pay one time, I don't understand, is OsmAND so unprofessional?
Who should I email?
I look forward to your reply, thank you in advance for your help.

Translated with DeepL.com (free version)

**Reply by Routhinator** [reply]
This answer isn't really an answer, as it does not cover how to subscribe without Google. I have a sub, but I see no way to maintain it once I remove Google play from all devices. The company needs to provide another way to subscribe.

**Reply by thaleb** [reply]
The answer is not valid (anymore). OSMAnd needs the playstore to verify your purchase. I am also getting rid of all Google related stuff but I can only see my purchases when the playstore is enabled, when disabled my Maps+ dissapears. Why are purchases not bound to your account? I wanted to get rid of Google Maps and other Google stuff and get on e/os/ later. But it's disappointing that OSMAnd forces you to use the playstore... Might as well cancel my subscription

**Reply by DmitryAlexei** [reply]
@thaleb are you created and logged in with your OsmAnd cloud account?
https://osmand.net/docs/user/personal/osmand-cloud?_highlight=cloud#osmand-start

**Reply by thaleb** [reply]
@DmitryAlexei Yes I am. I can sync and everything. But on the Purchases screen it only shows "OsmAnd Start", below that it says "Free account" and below that the purchase date when I purchased Maps+. But Maps+ only shows when the PlayStore is enabled.

**Reply by Zirochkabila** [reply]
@thaleb I am trying to tested your request, but I do not see any purchase screens in the F-Droid version (to verify the display of OsmAnd Start or Maps+). Please attach screenshots that would help us better understand your request.

<details><summary>Details</summary>
<img width="400" alt="Screenshot 2023-01-31 at 11 19 41" src="https://github.com/user-attachments/assets/5097e32c-11dc-4fe2-9f37-330efd47ec9e">
</details>

**Reply by thaleb** [reply]
My bad... I downloaded OSMAnd on Aurora. It does have a Purchases screen, but not showing Maps+ since that probably needs the playstore. So since I don't have the playstore now I paid for nothing i guess?

**Reply by Zirochkabila** [reply]
@thaleb I double-checked the display of Maps+ in OsmAnd downloaded from the Aurora Store - as you can see in the video, everything is displayed correctly.



<details><summary>Details</summary>
<video src="https://github.com/user-attachments/assets/d6ce88ae-829c-431d-9ba3-1c2ec4aa18a9"></video>
</details>

**Reply by thaleb** [reply]
@DmitryAlexei Is this with or without a enabled Play Store?

**Reply by Zirochkabila** [reply]
@thaleb Without Play Store

**Comment by mtalexan** [comment]
Is there any way to subscribe when you don't have anything that can run Android apps with any app stores?
I don't have access to any device that can support Google Store, Amazon Store,  or Huawei store for payment, nor an iPhone.

I think like many people, I would also encourage everyone to use an external sign up/payment service that doesn't take 30%, even if it wasn't (allowed to be) advertised in the app itself, if it was available.

**Comment by frederikb96** [comment]
> I use osmand~ from f-droid in /e/os (degoogled android), how can I use the cloud sync ?

Same for me. However, I used a virtual android machine on my pc to log into a Google account and buy the subscription via google play (You can also simply use another phone if you have one which has Google). After that I was able to log in with this email on my /e/OS phone where I dont have Google.
